projects/plugins/boost/app/assets/src/js/pages/settings/elements/ScoreBar.svelte

```diff
@@ -36,6 +42,10 @@
 			{:else}
 				<div class="jb-score-bar__score">{score}</div>
 			{/if}
+
+			{#if prevScore}
+				<div class="jb-score-bar__previous_score" style="left: min({prevScoreOffset}%, calc(100% - var(--clearence-space))">{prevScore}</div>
```

Comment:
🤔  Ok, I learned a new CSS trick here. Nice.

Is there an advantage to putting the previous score indicator inside the filler div instead of as a sibling to the filler div? It would save having to calculate the position relative to the filler div's width, and seems simpler to me. 😄

Reply:
It's just easier to calculate the clearance space. Otherwise the spacing doesn't calculate to a perfect value.